code/modules/computer3/computers/law.dm

```diff
@@ -60,6 +61,10 @@
 
 	attackby(obj/item/weapon/aiModule/module as obj, mob/user as mob, params)
 		if(istype(module, /obj/item/weapon/aiModule))
+			var/turf/T = get_turf(current)
+			if(T.z != z)
```

Comment:
Could this be changed to `atoms_share_level(T, src)`, as with above, to facilitate expansion of the space management system?